packages/pds/src/api/com/atproto/server/util.ts

```diff
@@ -117,3 +118,26 @@ const assertValidDocContents = async (
     )
   }
 }
+
+export const forwardIp: IForwardIp = <T extends HeadersParam>(
+  req: express.Request,
+  params: T | undefined,
+) => {
+  const ip = req.ip
+  const result: T | HeadersParam = params ?? { headers: {} }
+  if (ip) {
+    result.headers['x-forwarded-for'] = ip
+  }
```

Comment:
Wouldn't it be more accurate (in terms of semantics of that header), and easier, to simply add the `req.socket.remoteAddress` to the already provided list of `x-forwarded-for` ?

Reply:
Yeah that seems like a good call, should be resolved now.